keyv/Map.h

```diff
@@ -238,8 +238,8 @@ class Map
      * @param func callback function which is called for each found key
      * @version 1.14
      */
-    KEYV_API void getValues( const Strings& keys,
-                             const ConstValueFunc& func ) const;
+    KEYV_API void getValues( const Strings& keys, const ConstValueFunc& func )
```

Comment:
Parameters are together? Dunno, I don't really care much. I guess this was just motor memory.